pyproject.toml

```diff
@@ -167,6 +167,7 @@ snowflake = "ibis.backends.snowflake"
 spark = "ibis.backends.pyspark"
 sqlite = "ibis.backends.sqlite"
 trino = "ibis.backends.trino"
+adbc = "ibis.backends.adbc"
```

Comment:
You also need to add `adbc` to the `extras` section and declare its list of optional dependencies. Looking at your code, I think that's at least `adbc = ["pyarrow", "sqlalchemy"]` and possibly more if there's an ADBC dependency somewhere.